scripts/generate_train_scripts.py

```diff
@@ -7,25 +7,32 @@
 
 
 def main(
+    experiment_groups_path,
     experiment_group,
-    dataset_config_path,
+    dmpair_config_path,
     trainer_config_path,
     account_name,
     conda_env_path,
 ):
     # Dataset config
-    with open(dataset_config_path, "r") as stream:
-        dataset_config = yaml.safe_load(stream)
+    with open(
+        os.path.join(experiment_groups_path, experiment_group + ".yaml")
+    ) as stream:
+        experiment_group_config = yaml.safe_load(stream)
+
+    with open(dmpair_config_path, "r") as stream:
+        dmpair_config = yaml.safe_load(stream)
 
     # Prepare dataset list, seed list
-    NUM_SEEDS = len(dataset_config["seeds"])
-    NUM_PAIRS = len(dataset_config["experiment_groups"][experiment_group])
+    NUM_SEEDS = len(dmpair_config["seeds"])
+    NUM_PAIRS = len(experiment_group_config["dmpairs"])
 
     # Generate combinations of arguments to pass
     combinations = [
-        f"--dataset_config {dataset_config_path} "
-        + f"--trainer_config {trainer_config_path} "
+        f"--experiment_groups_path {experiment_groups_path} "
         + f"--experiment_group {experiment_group} "
+        + f"--dmpair_config {dmpair_config_path}"
```

Comment:
And again, space before the closing quotes?

Reply:
done